docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,241 @@
+          
+Then, fetch their remote fork and checkout the branch related to their PR:
+
+    git fetch their_fork
+    git checkout their_fork/branch_name
+
+Alternatively, to avoid being under the detached HEAD state, checkout and create a local branch with the command
+
+    git checkout -b branch their_fork/branch_name
+
+Follow the provided testing instructions to test their code on your VM. 
+
+If you made changes (with their permission), commit and push them to that branch:
+
+    git commit -m "Changes made"
+    git push their_fork branch_name
+
+Don’t forget to consult the Links (see above) for coding review standards and guidelines and the code review checklist.
+
+### Labels
+The `Passed Manual Tests` label can be added to a PR if a reviewer has tested the code on their own VM and determined that it performs as intended.
+
+The `Needs Rebase` label can be added if a PR has merge conflicts with the branch it is based off of.
+
+The `Needs Work` label can be added if a PR is not yet ready for review and requires more work before it can be approved.
+
+Other Labels can also be added by reviewers at their discretion. 
+
+### Code Review Etiquette
+The most important thing to remember during code review is to be respectful and positive in all correspondence with other developers. It is also good practice
+to be on top of all comments or requests and to respond to every comment.
+
+#### When to ask for or remind someone to do a review
+You may ask for someone to review your PR at your discretion. However, do not assign someone to your PR.
+
+In general, if someone has previously reviewed your PR and requested changes, it is good practice to re-request their review once you have implemented those changes.
+
+If you have requested review on a PR and have not received feedback for a number of days, it is okay to either re-request their review or leave a comment
+on the PR pinging the developer with a short message. For example, "Hey @developer, could you please review this PR and let me know what you think."
+
+#### Implementing suggestions or requested changes
+It is good practice to implement any requested changes as soon as possible and then re-request review from the developer. It also helps the team if you acknowledge each comment or request with a few words or just an emoji, so that anyone reading will know that you've seen it.
+
+If you feel that the suggested or requested change is not necessary, feel free to respond in the comment section with a clear explanation of why you don't think it is necessary.
+Make sure to stay positive and respectful in all review or comment threads.
+
+#### When to resolve a review or comment thread
+If you have implemented a requested change from a review-comment thread, it is good practice to let the developer who requested the change
+resolve the thread. However, if a comment thread is simple or straightforward and has been resolved, feel free to resolve it.
```

Comment:
```suggestion
resolve the thread, so that they get the chance to check your change met their expectations. However, if a comment thread is simple or straightforward (e.g. a typo), feel free to resolve it.
```